modules/drivers/sqlserver/project.clj

```diff
@@ -1,8 +1,8 @@
-(defproject metabase/sqlserver-driver "1.0.0-SNAPSHOT-7.4.1.jre8"
```

Comment:
you should also update it in `sqlserver/resources/plugin-manifest.yaml`. That's the version information that is actually available to Metabase at runtime. We're not doing anything with version information at the moment, but at some point in the future we might include that as debugging information or something

Reply:
Done.